rosdep/python.yaml

```diff
@@ -1002,7 +1002,9 @@ python-cherrypy:
   debian: [python-cherrypy3]
   fedora: [python-cherrypy]
   gentoo: [dev-python/cherrypy]
-  ubuntu: [python-cherrypy3]
+  ubuntu:
+    '*': [python-cherrypy3]
+    focal: [python3-cherrypy3]
```

Comment:
To support python3, please leave the existing `python-` keys (which are implicitly Python 2), and instead add new `python3-` keys.  That goes for all of the ones below as well.

Reply:
Ah, sorry about that. I've updated it now